app/views/shared/_flashes.html.slim

```diff
@@ -0,0 +1,3 @@
+- flash.each do |name, message|
+  = content_tag :div do
+    = content_tag :span, message.html_safe, class: ['flash', name]
```

Comment:
This could be dangerous because you're accepting the entire message as `html_safe`. For example if something unsafe is in the `@lesson.course.name`, then this will let it through (plus potentially other user-controlled content in the future, whenever you're using a flash message). Try only to accept the result of `link_to` as `html_safe`.

Reply:
So, it seems the `html_safe` can be used just on in the view, per [this comment](https://stackoverflow.com/questions/43358309/im-trying-to-implement-link-to-in-a-flash-message-for-an-update-action-in-a-c). 

So I created a special notice for messages which contain a url.